roles/apply_nmstate/tasks/main.yml

```diff
@@ -1,21 +1,22 @@
-- name: Install nmstate
-  package:
-    name: nmstate
-    state: present
-
-- name: "Copy rendered_nmstate_yml to {{ vm_nmstate_config_path }}"
-  copy:
-    content: "{{ rendered_nmstate_yml }}"
-    dest: "{{ vm_nmstate_config_path }}"
-    mode: 0644
-
-  # No commit is done to revert the changes if they cause the host to be come unreachable
-- name: Apply nmstate
-  shell: # noqa: no-changed-when
-    cmd: "nmstatectl apply --no-commit --timeout 120 {{ vm_nmstate_config_path }}"
-  async: 60
-  poll: 5
-
-- name: Commit changes
-  shell: # noqa: no-changed-when
-    cmd: "nmstatectl commit"
+---
+- name: Applying Nmstate
+  block:
+    - name: Install nmstate
+      package:
+        name: nmstate
+        state: present
+    - name: "Copy rendered_nmstate_yml to {{ vm_nmstate_config_path }}"
+      copy:
+        content: "{{ rendered_nmstate_yml }}"
+        dest: "{{ vm_nmstate_config_path }}"
+        mode: 0644
+      # No commit is done to revert the changes if they cause the host to be come unreachable
+    - name: Apply nmstate
+      shell: # noqa: no-changed-when
+        cmd: "nmstatectl apply --no-commit --timeout 120 {{ vm_nmstate_config_path }}"
+      async: 60
+      poll: 5
+    - name: Commit changes
+      shell: # noqa: no-changed-when
+        cmd: "nmstatectl commit"
```

Comment:
We should keep the newlines that used to separate these tasks. In my opinion, YAML squished in this way is quite difficult to read.

Reply:
I think _blocks_ are hard to read as it is. 👍